core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -854,6 +864,10 @@ static function (array $allowedProperties): array {
                                     if (false === @preg_match(WysiwygStyleProcessor::prepareRegex($regex), '')) {
                                         throw new \InvalidArgumentException(sprintf('The regex "%s" for property "%s" is invalid.', $regex, $property));
                                     }
+
+                                    if (str_contains($regex, '.*')) {
+                                        throw new \InvalidArgumentException(sprintf('The regex "%s" for property "%s" contains ".*" which is not allowed due to security reasons.', $regex, $property));
+                                    }
```

Comment:
Is this too strict?
If someone knows what they are doing they can bypass it using `[^;]*` instead.